### PR TITLE
Make dependancy to hpp-wholebody-step-corba optionnal

### PR DESCRIPTION
**Pull request by pFernbach**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -58,14 +58,17 @@ ENDIF()
...
+ADD_OPTIONAL_DEPENDENCY("hpp-wholebody-step-corba >= 3")
 IF (NOT CLIENT_ONLY)
   ADD_DOC_DEPENDENCY("hpp-core >= 3")
   ADD_REQUIRED_DEPENDENCY("hpp-manipulation")
   ADD_REQUIRED_DEPENDENCY("hpp-manipulation-urdf")
 ENDIF (NOT CLIENT_ONLY)
 ADD_REQUIRED_DEPENDENCY("omniORB4 >= 4.1.4")
 
+SET(HPP_MANIPULATION_HAS_WHOLEBODY_STEP ${HPP_WHOLEBODY_STEP_FOUND}
+  CACHE BOOL "Compile with dependency to hpp-wholebody-step")
```
Comment (florent-lamiraux) [inline CMakeLists.txt]: Can you set default value to TRUE so that default behavior is not changed ?

**Comment by florent-lamiraux** [comment]
Can you set default value of cmake cache variable to TRUE in order to keep the default behavior unchanged ?
Making the configuration depend on what is detected or not is misleading for the user.

**Comment by pFernbach** [comment]
Done. But the same behavior is used in hpp-manipulation (https://github.com/humanoid-path-planner/hpp-manipulation/blob/master/CMakeLists.txt).

**Comment by florent-lamiraux** [comment]
Thank you.
I did not notice. I think it should be changed.

**Pull request merged**